### PR TITLE
pacific: qa/tasks/cephadm: enable mon_cluster_log_to_file

### PR DESCRIPTION
**Pull request by adk3798**
Backport of https://github.com/ceph/ceph/pull/41479 and https://github.com/ceph/ceph/pull/54312.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by idryomov** [comment]
Nothing that this also includes https://github.com/ceph/ceph/pull/41479.

**Review by idryomov — APPROVED** [review]

**Comment by ljflores** [comment]
I reviewed the PR. This did cause some new red jobs to appear in the rados suite, but most are warnings that are expected in test environments, and thus should be whitelisted. I raised the following tracker tickets to address the whitelists (although I don’t think it should block the merge of this PR):
https://tracker.ceph.com/issues/64343
https://tracker.ceph.com/issues/64344

Regarding map eXX had wrong heartbeat addr, it looks like the same issue as https://tracker.ceph.com/issues/58915 . I have asked @radoslawzarzynski to take a look and see if the fix needs to be included in pacific.

Finally , I noticed a cephadm bug that looks new. @adk3798  wdyt of https://tracker.ceph.com/issues/53154 ? It was caught when the job failed from CEPHADM_STRAY_DAEMON in the cluster log. Is it a true bug or should we whitelist that warning?

In summary, everything that needs to be whitelisted now in the rados suite has a tracker, so I am okay with merging this PR since it has already merged to main.

Rados approved; full summary: https://tracker.ceph.com/projects/rados/wiki/PACIFIC#httpstrellocomcERqmvaZu1947-wip-yuri10-testing-2024-02-02-1149-pacific
